WalletWasabi.Gui/Controls/WalletExplorer/ClosedWalletViewModel.cs

```diff
@@ -0,0 +1,60 @@
+using AvalonStudio.Extensibility;
+using ReactiveUI;
+using Splat;
+using System.IO;
+using System.Linq;
+using System.Reactive;
+using System.Threading;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class ClosedWalletViewModel : WalletViewModelBase
+	{
+		public ClosedWalletViewModel(string path) : base(Path.GetFileNameWithoutExtension(path))
+		{
+			OpenWalletCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				IsBusy = true;
+
+				var global = Locator.Current.GetService<Global>();
+
+				if (!await global.WaitForInitializationCompletedAsync(CancellationToken.None))
+				{
+					return;
+				}
+
+				var walletManager = global.WalletManager;
+
+				var keyManager = global.LoadKeyManager(Title);
```

Comment:
This method not only can return null but also throws exceptions.

Reply:
done